index.d.ts

```diff
@@ -1668,7 +1682,7 @@ declare namespace Eris {
     executeWebhook(webhookID: string, token: string, options: WebhookPayload): Promise<void>;
     followChannel(channelID: string, webhookChannelID: string): Promise<ChannelFollow>;
     getBotGateway(): Promise<{ session_start_limit: { max_concurrency: number; remaining: number; reset_after: number; total: number }; shards: number; url: string }>;
-    getChannel(channelID: string): AnyChannel;
+    getChannel<T extends Channel = AnyChannel>(channelID: string): T;
```

Comment:
We discussed this a couple times, concluded that type assertions are more appropriate